rundeckapp/grails-app/views/framework/_createProjectForm.gsp

```diff
@@ -82,7 +82,6 @@
 
     <asset:javascript src="framework/editProject.js"/>
     <asset:javascript src="static/pages/project-config.js" defer="defer" />
-    <asset:stylesheet href="static/css/pages/project-config.css" />
```

Comment:
This file isn't being created, as the respective components do not have any css to be imported.